### DISCUSSION TITLE
Has picker never supported filter options?

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#2494] [Q&A] [answered]
I was answering a doodles question on reddit earlier today and went the easy route of using element picker to propose the solution. I was kinda surprised the filter wouldn't match when I added the `$image` option.

I then used the logger, and it proposed the same option, with the addition of the domain too. I copy-pasted it to picker to double-check I didn't make any typos and the filter wouldn't match either.

Was that always a thing? I don't use network-filtering in picker often, but I really thought the filter options used to be supported.  
I went back a few uBO versions and didn't fine one where it worked though.

So is this by design? Was this discussed somewhere before? I couldn't find any issues or discussions about this. It's also not documented on element picker's page.

**Comment by gorhill** [comment, marked as answer]
The element picker's text area does not compile into a static network filter, it just tries to find URL matches according to the pattern -- filter options will be interpreted as pattern to match.
